src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -76,6 +94,9 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode ||
+                DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && ($isVendor = self::inVendors($file));
```

Comment:
should be on a single line - please remove the bracket arount the assignation, but add some around the logical operators